### PR TITLE
Adding s3 bucket, sys test and unit test

### PR DESCRIPTION
**Pull request by fuddstar**
Build Passing 
https://gaautobots.atlassian.net/builds/browse/AM-AUT12-4

### CODE REVIEW

File: amazonia/classes/s3.py [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+        """
+        AWS - http://docs.aws.amazon.com/AWSCloudFormation/latest/UserGuide/aws-properties-s3-policy.html
+        Troposphere - https://github.com/cloudtools/troposphere/blob/master/troposphere/s3.py
+        s3_access = [Private,
+                    PublicRead,
+                    PublicReadWrite,
+                    AuthenticatedRead,
+                    BucketOwnerRead,
+                    BucketOwnerFullControl,
+                    LogDeliveryWrite]
```
Comment (snayler0) [inline amazonia/classes/s3.py]: params?

File: test/sys_tests/test_sys_s3.py [hunk 2]
```diff
@@ -0,0 +1,18 @@
+from troposphere import Template
+
+from amazonia.classes.s3 import S3
+
+
+def main():
+    template = Template()
```
Comment (snayler0) [inline test/sys_tests/test_sys_s3.py]: docstring here?

File: amazonia/classes/s3.py (hunk 1 above)
Comment (fuddstar) [inline amazonia/classes/s3.py]: added

**Comment by snayler0** [comment]
Approved

**Pull request merged**